src/extensions/as-legacy-globals-for-extension-api/legacy-global-di-for-extension-api.ts

```diff
@@ -3,6 +3,7 @@
  * Licensed under MIT License. See LICENSE in root directory for more information.
  */
 import type { DiContainer } from "@ogre-tools/injectable";
```

Comment:
Changes in this file are not needed. There is no functionality changes and usage of iter just makes it more complex.